types/component/ChatList/ChannelImage.types.ts

```diff
@@ -2,18 +2,19 @@
 
 import {ImageStyle} from 'react-native-fast-image';
 import {StyleProp} from 'react-native';
+import {BetterSocialChannelType} from '../../database/schema/ChannelList.types';
 
 export type ChannelImageProps = React.FC & {
   Big: React.FC<ChannelImageMainProps>;
   Small: React.FC<ChannelImageBadgeProps>;
 };
 
 export type ChannelImageMainProps = {
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: BetterSocialChannelType;
   image?: string;
   style?: StyleProp<ImageStyle>;
 };
 
 export type ChannelImageBadgeProps = {
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: BetterSocialChannelType;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the change from using string literals to an enum for the `type` property in `ChannelImageBadgeProps` is also a good practice. It provides better type safety and makes the code more maintainable.

```diff
-  type: 'COMMUNITY' | 'GROUP' | 'GROUP_INFO';
+  type: BetterSocialChannelType;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->